### PR TITLE
refactor: update configs folder

### LINKED ISSUE #1033: [FEATURE]: Reduce config files
### Tthe purpose of the feature:
Integrate the systems' logger default into a single yaml file and move the **common** training controlling flags from systems files into the arch yaml files (e.g. `seed` and `total_timesteps`). This helps avoid setting things in each system in case we want to run a sweep for example and avoid adding a logger_default for each system.

### Describe the solution you'd like
Loggers merged in one file and  setting the common training flags outside each system 😄 

### How do we know when the implementation of this feature is complete?
Checklist:
- [x] Merge the logger files into one yaml file and add the `system_name` in the logger.py code by using the `tags.append(system.system_name)`.
- [ ] Add the common items related to training all systems in the arch files such as seed, total_timesteps, and num_updates.

### Additional context
Check this PR #987 for some inspiration (An example was started in this PR but for now, we keep the systems' default separate) 🙌 

🌟 Additional request: Merge system defaults for the same system type (such PPOs) and find the best way to select the system config without changing the default.

### PR DESCRIPTION
**Pull request by WiemKhlifi**
## What?
Updating the configs folder.

## How?
* Storing all defaults in one folder for Anakin systems.
* Removing all system_name_logger and use one base_logger.
* Update the config paths in systems & test files.

### CODE REVIEW

File: test/integration_test.py [hunk 1]
```diff
@@ -23,6 +23,8 @@
 # system run all envs, but each env and each system is run at least once.
 # For each system we select a random environment to run.
 # Then for each environment we select a random system to run.
+anakin_config_path = "../mava/configs/default/anakin"
+
 ppo_systems = [
     "ppo.anakin.ff_ippo",
```
Comment (WiemKhlifi) [inline test/integration_test.py]: Here I'm thinking we could do something like the below when we have all the systems for the sebulba arch else we do them separately:
```suggestion
arch="anakin"
config_path = f"../mava/configs/default/{arch}"
ppo_systems = [
    f"ppo.{arch}.ff_ippo",
```

**Review by sash-a — CHANGES_REQUESTED** [review]
Thanks Wiem!

Couple small things:
1. Just put the default configs in `configs/default` (no anakin) anakin/sebulba will share the same configs except for arch to start. If we find that params differ a lot then we can change it later. Because of this I think you can leave the integration tests as is
2. I think remove the `default_...` part of the config file name because they're now in the default folder.

**Comment by WiemKhlifi** [comment]
Thank you Sasha for these suggestions! Without the `default_` is definitely cleaner I made all the necessary changes please check them 🙏

File: mava/advanced_usage/ff_ippo_store_experience.py
Comment (OmaymaMahjoub) [inline mava/advanced_usage/ff_ippo_store_experience.py:1]: if you can add `cfg.logger.system_name = "ff_ippo"` to this one as well

**Review by OmaymaMahjoub — CHANGES_REQUESTED** [review]
Thanks @WiemKhlifi  for the changes, I added minor request 🙏

File: mava/advanced_usage/ff_ippo_store_experience.py
Comment (WiemKhlifi) [reply mava/advanced_usage/ff_ippo_store_experience.py:1]: Nice catch thanks 🙌

File: mava/advanced_usage/ff_ippo_store_experience.py [hunk 2]
```diff
@@ -670,11 +670,12 @@ def _reshape_experience(experience: Dict[str, chex.Array]) -> Dict[str, chex.Arr
     logger.stop()
 
 
-@hydra.main(config_path="../configs", config_name="default_ff_ippo.yaml", version_base="1.2")
+@hydra.main(config_path="../configs/default", config_name="ff_ippo.yaml", version_base="1.2")
```
Comment (OmaymaMahjoub) [inline mava/advanced_usage/ff_ippo_store_experience.py:673]: default/  you need to add "/"

Comment (WiemKhlifi) [reply mava/advanced_usage/ff_ippo_store_experience.py:673]: It works with or without but nice catch it makes all files the same 🙌

Comment (WiemKhlifi) [reply mava/advanced_usage/ff_ippo_store_experience.py:673]: Well I guess I'll do the opposite and remove the '/' from all files 😅

Comment (OmaymaMahjoub) [reply mava/advanced_usage/ff_ippo_store_experience.py:673]: oh I see!

**Review by OmaymaMahjoub — DISMISSED** [review]
LGTM! 🔥

**Review by sash-a — DISMISSED** [review]
Thanks Wiem! Looks great :medal_sports:

**Review by OmaymaMahjoub — APPROVED** [review]
Thanks @WiemKhlifi 🙏

**Review by sash-a — APPROVED** [review]

**Pull request merged**